Libraries/src/Amazon.Lambda.Annotations/LambdaOutputExecutableAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿
+using System;
+
+namespace Amazon.Lambda.Annotations
+{
+    /// <summary>
+    /// Indicates this the Lambda function is going to target an executable instead of a class based handler.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
```

Comment:
I can add it as an assembly attribute, but I didn't think it was possible to access an assembly attribute in a source generator because the assembly doesn't exist yet